prow/kube/prowjob.go

```diff
@@ -79,6 +79,8 @@ type ProwJobStatus struct {
 	URL             string       `json:"url,omitempty"`
 	PodName         string       `json:"pod_name,omitempty"`
 	BuildID         string       `json:"build_id,omitempty"`
+	UUID            string       `json:"uuid,omitempty"`
```

Comment:
I understand now, we need better terminology for these, I've been conflating job name (`pull-kubernetes-bazel`) and these fields (UUIDs) a lot when parsing your comments. I'll rewrite this shortly to skip the base path and just provide a UUID / parent UUID in ENV. :smiley: